mlem/runtime/server.py

```diff
@@ -19,6 +21,14 @@ class Config:
     abs_name: ClassVar[str] = "server"
     env_vars: ClassVar[Optional[Dict[str, str]]] = None
     additional_source_files: ClassVar[Optional[List[str]]] = None
+    request_serializer: Optional[Serializer] = None
```

Comment:
```suggestion
    request_serializer: Optional[str, Serializer] = None
```

Reply:
Since it works like 
```python
   ...: serve("get-started/sklearn", FastAPIServer(
   ...:         request_serializer="file",
   ...:         response_serializer="file"
   ...: ))
```

Reply:
Nope, in my example only `FastAPIServer(request_serializer=FileSerializer())` is allowed. It's the same for all other cases when you provide an instance of server/builder/client/etc - it's nested fields should be instances too.
You are probably confused by the ability to provide strings instead of instances for topmost objects. In this case you also should provide kwargs, and those can have nested strings. Thats because  the topmost object is not constructed in this case and those nested fields will be constructed together with it. But if you already have topmost object, this logic would not work

Reply:
you can try this - it works
```python
from mlem.api import serve
from mlem.contrib.fastapi import FastAPIServer
from mlem.core.data_type import FileSerializer

serve("sklearn", FastAPIServer(
    request_serializer=FileSerializer(),
    response_serializer=FileSerializer()
))
```
or maybe be I got you wrong?

Reply:
This works
```python
FastAPIServer(
    request_serializer=FileSerializer(),
    response_serializer=FileSerializer()
)
```
This does not
```python
FastAPIServer(
    request_serializer="file",
    response_serializer="file"
)
```

Reply:
Sorry, that was a typo in my last comment. This works for me:
<img width="870" alt="image" src="https://user-images.githubusercontent.com/6797716/201316522-1f823df6-2637-41ce-ba70-c024f343d2a1.png">